src/cmap/handshake/handshake_document.ts

```diff
@@ -0,0 +1,18 @@
+import type { Document } from 'bson';
+
+import type { ClientMetadata } from './client_metadata';
+
+/** @internal */
+export interface HandshakeDocument extends Document {
+  /**
+   * @deprecated Use hello instead
+   */
+  ismaster?: boolean | 0 | 1;
+  hello?: boolean | 0 | 1;
+  helloOk?: boolean;
```

Comment:
We are a hello capable driver so we should always send helloOk, could we make this required?